selfdrive/ui/soundd.cc

```diff
@@ -21,67 +21,51 @@ void sigHandler(int s) {
 
 class Sound : public QObject {
 public:
-  explicit Sound(QObject *parent = 0) {
+  explicit Sound(QObject *parent = 0) : sm({"carState", "controlsState"}) {
     // TODO: merge again and add EQ in the amp config
     const QString sound_asset_path = Hardware::TICI() ? "../assets/sounds_tici/" : "../assets/sounds/";
     std::tuple<AudibleAlert, QString, bool> sound_list[] = {
-      {AudibleAlert::CHIME_DISENGAGE, sound_asset_path + "disengaged.wav", false},
-      {AudibleAlert::CHIME_ENGAGE, sound_asset_path + "engaged.wav", false},
-      {AudibleAlert::CHIME_WARNING1, sound_asset_path + "warning_1.wav", false},
-      {AudibleAlert::CHIME_WARNING2, sound_asset_path + "warning_2.wav", false},
-      {AudibleAlert::CHIME_WARNING2_REPEAT, sound_asset_path + "warning_2.wav", true},
-      {AudibleAlert::CHIME_WARNING_REPEAT, sound_asset_path + "warning_repeat.wav", true},
-      {AudibleAlert::CHIME_ERROR, sound_asset_path + "error.wav", false},
-      {AudibleAlert::CHIME_PROMPT, sound_asset_path + "error.wav", false}
-    };
+        {AudibleAlert::CHIME_DISENGAGE, "disengaged.wav", false},
+        {AudibleAlert::CHIME_ENGAGE, "engaged.wav", false},
+        {AudibleAlert::CHIME_WARNING1, "warning_1.wav", false},
+        {AudibleAlert::CHIME_WARNING2, "warning_2.wav", false},
+        {AudibleAlert::CHIME_WARNING2_REPEAT, "warning_2.wav", true},
+        {AudibleAlert::CHIME_WARNING_REPEAT, "warning_repeat.wav", true},
+        {AudibleAlert::CHIME_ERROR, "error.wav", false},
+        {AudibleAlert::CHIME_PROMPT, "error.wav", false}};
```

Comment:
revert the indentation change and put the closing bracket on a new line like before